platform-tests/src/availability/healthcheck_availability_during_deployment_test.go

```diff
@@ -45,7 +45,8 @@ func errorRateThreshold(metrics *vegeta.Metrics, minimumTestDuration time.Durati
 	metrics.Close()
 
 	fmt.Printf(
-		" - Duration: %s, Requests: %d, Non 200 Requests: %d, Success Rate: %.2f%%\n",
+		"%s: Duration: %s, Requests: %d, Non 200 Requests: %d, Success Rate: %.2f%%\n",
```

Comment:
Something that bugged me is that the timestamps aren't always the same length which makes it harder to match the other fields across multiple lines:
```
2017-01-17 12:01:07.320338954 +0000 UTC: Duration: 6m49.99999862s, Requests: 4101, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:12.320654329 +0000 UTC: Duration: 6m54.99999862s, Requests: 4151, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:17.32094242 +0000 UTC: Duration: 6m59.99999862s, Requests: 4201, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:22.32122625 +0000 UTC: Duration: 7m4.99999862s, Requests: 4251, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:27.321503754 +0000 UTC: Duration: 7m9.99999862s, Requests: 4301, Non 200 Requests: 0, Success Rate: 100.00%
2017-01-17 12:01:32.321791137 +0000 UTC: Duration: 7m14.99999862s, Requests: 4351, Non 200 Requests: 0, Success Rate: 100.00%
```